src/org/benf/cfr/reader/bytecode/analysis/variables/NamedVariableMethodParameter.java

```diff
@@ -0,0 +1,49 @@
+package org.benf.cfr.reader.bytecode.analysis.variables;
+
+import org.benf.cfr.reader.bytecode.analysis.types.MethodPrototype;
+import org.benf.cfr.reader.entities.attributes.MethodParameterEntry;
+import org.benf.cfr.reader.util.output.Dumper;
+
+import java.util.List;
+
+public class NamedVariableMethodParameter implements NamedVariable {
+    private String name;
+    private boolean forced = false;
+    private List<MethodParameterEntry> parameters;
+
+    public NamedVariableMethodParameter(List<MethodParameterEntry> parameters, String name) {
+        this.name = name;
+        this.parameters = parameters;
+    }
+
+    @Override
+    public void forceName(String name) {
+        this.name = name;
+        forced = true;
+    }
+
+    @Override
+    public String getStringName() {
+        return this.name;
+    }
+
+    @Override
+    public boolean isGoodName() {
+        return true;
```

Comment:
Same here, no good if it's 0-name; it's totally legal to have null name in MethodParameters (storing parameter flags only) but valid name in LocalVariableTable/LocalVariableTypeTable

Reply:
not sure what exactly isGoodName is supposed to represent, but I don't see a very easy way to fix this currently since this doesn't know which parameter it is until dumpParameter is called...